build/scripts/previousversion/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	semver "github.com/blang/semver/v4"
+)
+
+func main() {
+	baseVersion := flag.String("version", "", "the base version to be taken in as input for release")
+	flag.Parse()
+	v, _ := semver.Parse(*baseVersion)
+	releaseVersion := fmt.Sprintf("%d.%d.%d", v.Major, v.Minor-1, v.Patch)
```

Comment:
ooooh! So we missed a small part of the requirements in the ticket:

> (replace . with -)

But we can do this here! Use `-` rather than `.`, and we are good to go!

Reply:
Updated it.